### ISSUE TITLE
@suppress for CTOR_IN_DIFFERENT_SCOPE

### ISSUE THREAD

**Issue by bugreportuser** [google/closure-compiler#2870]
The warning for [CTOR_IN_DIFFERENT_SCOPE](https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/GlobalTypeInfoCollector.java#L100) cannot be suppressed, so the warning for this code can't be removed:

```javascript
(function() {
    Array.prototype.a = function() {};
})();
```

```
> java -jar closure-compiler-v20180319.jar --new_type_inf test.js
test.js:2: WARNING - Modifying the prototype is only allowed if the constructor is in the same scope

    Array.prototype.a = function() {};
    ^^^^^^^^^^^^^^^^^

0 error(s), 1 warning(s), 100.0% typed
(function(){Array.prototype.a=function(){}})();
```

Can a warning group be added for this?

This warning only happens for the new type inference.

**Comment by lauraharker** [comment]
Closing since [NTI is deprecated](https://github.com/google/closure-compiler/wiki/%5BOBSOLETE%5D-Using-NTI-(new-type-inference))
